src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-service-pop-up/ubs-admin-tariffs-add-service-pop-up.component.html

```diff
@@ -27,6 +30,9 @@ <h4>{{ 'ubs-tariffs-add-service.edit_service' | translate }}</h4>
           id="nameEng"
           placeholder="{{ 'ubs-tariffs-add-service.placeholder-service' | translate }}"
         />
+        <span class="validation" *ngIf="addServiceForm.controls['nameEng'].errors">{{
+          'ubs-tariffs-add-service.name-error' | translate
```

Comment:
better to use here input-error-component

Reply:
i fixed it